### PR TITLE
Archive page

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds the ArchivePage into Notes taking app. Users can archive/unarchive the notes. 

Other features added are
- Notes can be deleted from homepage or archive page.
- Time of note creation has been added.
- Notes can be labelled on 4 categories.
- Sort notes by time of creation has been added.

[live preview](https://deploy-preview-4--notestalk.netlify.app/)
Would appreciate you feedback and suggestions.

image

image

### CODE REVIEW

**Review by Rockyraven — APPROVED** [review]
your code is clean and easy to readable 
the way of importing component is really impreesive

File: src/context/note-context.js [hunk 1]
```diff
@@ -2,21 +2,49 @@ import axios from "axios";
...
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+      ];
```
Comment (Shailseen) [inline src/context/note-context.js]: I can't find the use of this array. Can you remove this if this one is uselsess?

File: src/context/note-context.js [hunk 2]
```diff
@@ -92,8 +148,137 @@ const NoteProvider = ({ children }) => {
...
+      })();
+    } else {
+      showToast({ message: "Login to Delete notes", type: "failure" });
+    }
+  }
+
+  function sortNewToOld() {
+    const temp = notesList;
+    temp.sort((a, b) => new Date(a.dateTime) - new Date(b.dateTime));
+    setNotesList(() => temp);
```
Comment (Shailseen) [inline src/context/note-context.js:240]: I think you can directly use notesList for sorting the data.

**Review by Shailseen — COMMENTED** [review]
You have to only consider on the comments, otherwise the folder structuring and code is good.
And way of giving PR details is also very decent & understandable.

File: src/context/note-context.js (hunk 1 above)
Comment (vishalg8454) [reply src/context/note-context.js]: Ah yes, removing it.

File: src/context/note-context.js (hunk 2 above)
Comment (vishalg8454) [reply src/context/note-context.js:240]: okay, will consider this.

**Pull request merged**